<?xml version="1.0"?>

<launch>

  <!-- moveit_task_constructor_demo -->
  <node name="moveit_task_constructor_demo2" pkg="moveit_task_constructor_demo" type="moveit_task_constructor_demo2" output="screen">
    <param name="execute" value="true" />
    <rosparam command="load" file="$(find moveit_task_constructor_demo)/config/panda_config2.yaml" />
  </node>

</launch>